presto-main/src/main/java/com/facebook/presto/operator/window/ReflectionWindowFunctionSupplier.java

```diff
@@ -32,35 +32,49 @@
         extends AbstractWindowFunctionSupplier
 {
     private final Constructor<T> constructor;
+    private final boolean canIgnoreNulls;
 
     public ReflectionWindowFunctionSupplier(String name, Type returnType, List<? extends Type> argumentTypes, Class<T> type)
     {
-        this(new Signature(name, WINDOW, returnType.getTypeSignature(), Lists.transform(argumentTypes, Type::getTypeSignature)), type);
+        this(new Signature(name, WINDOW, returnType.getTypeSignature(), Lists.transform(argumentTypes, Type::getTypeSignature)), false, type);
     }
 
-    public ReflectionWindowFunctionSupplier(Signature signature, Class<T> type)
+    public ReflectionWindowFunctionSupplier(String name, Type returnType, boolean canIgnoreNulls, List<? extends Type> argumentTypes, Class<T> type)
```

Comment:
The constructor above is only used for testing, so add another parameter to it rather than adding another constructor.